### PR TITLE
[NO-JIRA] Fix accordion README

### PR DESCRIPTION
**Pull request by georgegillams**
Remember to include the following changes:
+ [x] `UNRELEASED.yaml`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

### CODE REVIEW

File: UNRELEASED.yaml [hunk 1]
```diff
@@ -27,4 +30,4 @@
 
 ADDED:
```
Comment (ojcurtis) [inline UNRELEASED.yaml]: Ooops looks like I didn't update the changelog after I released yesterday 😂

File: packages/bpk-component-accordion/README.md [hunk 2]
```diff
@@ -71,14 +71,15 @@ export default () => (
...
-| tagName  | string   | false    | span          |
+| Property  | PropType | Required | Default Value |
+| --------- | -------- | -------- | ------------- |
+| children  | node     | true     | -             |
+| id        | string   | true     | -             |
+| title     | string   | true     | -             |
+| expanded  | bool     | false    | false         |
+| onClick   | func     | false    | () => null    |
+| tagName   | string   | false    | span          |
+| textStyle | string   | false    | null          |
```
Comment (ojcurtis) [inline packages/bpk-component-accordion/README.md]: The default value mentioned here doesn't match what is in the code. Can this be changed to `base`

Comment (georgegillams) [reply packages/bpk-component-accordion/README.md]: Oh dope!

File: UNRELEASED.yaml (hunk 1 above)
Comment (georgegillams) [reply UNRELEASED.yaml]: Do you want me to remove this from `UNRELEASED.md` then?

Comment (ojcurtis) [reply UNRELEASED.yaml]: Nah we can just have it inserted into the changelog when we do the release 😄

Comment (georgegillams) [reply UNRELEASED.yaml]: 👍

**Pull request merged**